inference/core/workflows/core_steps/loader.py

```diff
@@ -3,144 +3,140 @@
 from inference.core.cache import cache
 from inference.core.env import API_KEY, WORKFLOWS_STEP_EXECUTION_MODE
 from inference.core.workflows.core_steps.common.entities import StepExecutionMode
-from inference.core.workflows.core_steps.flow_control.continue_if.version_1 import (
+from inference.core.workflows.core_steps.flow_control.continue_if.v1 import (
```

Comment:
this is purely convention right?  not a blocker but wondering if having the default be v1.py in folder might make it slightly less convenient to look / search for files (editor will have lots of v1.py files open e.g.)

Reply:
Previously we had `version_1.py`, changed as I liked this suggestion https://github.com/roboflow/inference/pull/565#discussion_r1709258820

I believe some structure is needed vs coming up with the name each and every time - what u said about IDE inconvenience is true, but here we need to make some choice to set pattern that would be scalable - do you have any other suggestion? I am happy to change

Reply:
I think `from inference.core.workflows.core_steps.flow_control.continue_if.v1` looks nice when it comes to import. Users can still add `import .foo_bar as v1` into block init

Reply:
I also like that